setup.py

```diff
@@ -29,6 +32,8 @@
     license="AGPLv3",
```

Comment:
Can we remove the `install_requires` for `pymongo[srv]` and `psycopg2-binary` too?

Alternatively, borrow this: https://github.com/overhangio/tutor/blob/master/setup.py#L60